lib/src/store.rs

```diff
@@ -149,18 +151,35 @@ impl Store {
         id: &ConflictId,
     ) -> BackendResult<conflicts::Conflict<Option<TreeValue>>> {
         let backend_conflict = self.backend.read_conflict(path, id)?;
-        Ok(conflicts::Conflict::from_backend_conflict(
-            &backend_conflict,
-        ))
+        let removes = backend_conflict
```

Comment:
Is prefer to keep this in the `from_backend_conflict` function, if you don't mind. OTOH, it could take a Conflict instead of a reference.

Reply:
Also, it could, in principle, belong to backend::Conflict, if you prefer conflicts.rs to be unaware of the backend.

Reply:
Sure, I don't mind having the `from_backend_conflict()` function. I'm fine with the dependency on `backend`.